doc/repro.rst

```diff
@@ -11,9 +11,144 @@ Elsewhere:
+- For *ongoing* work, the documentation **must** allow colleagues to locate and understand the state of current and planned work.
+
+- Every distinct project for which MESSAGEix-GLOBIOM scenarios and outputs are used **must** be included in the documentation.
+  The contents **may** be brief or extensive; read on.
+
+- Docs **must** be placed in one of the following locations:
+
+  - :file:`doc/model/{variant}.rst`, or :file:`doc/model/{variant}/index.rst`, or :file:`doc/{variant}/index.rst` if there will be multiple documentation pages for the model variant.
+  - :file:`doc/project/{name}.rst`, or :file:`doc/project/{name}/index.rst` or :file:`doc/{name}/index.rst` if there will be multiple documentation pages for the project.
+
+  In :mod:`message_data`, some docs have been placed ‘inline’ with the code, for example in:
+
+  - :file:`message_data/model/{variant}/doc.rst`
+  - :file:`message_data/model/{variant}/doc/index.rst`
+  - :file:`message_data/project/{name}/doc.rst`
+  - :file:`message_data/project/{name}/doc/index.rst`
+
+  When code is :doc:`migrated <migrate>` from :mod:`message_data`, these files **should** be moved to the :file:`/doc/` directory.
+
+- One file, usually the main or index file **must** be included in the :code:`.. toctree::` directive in :file:`doc/index.rst`.
+
+- Extensive documentation for a project or model variant **should** be organized with headings, tables of contents, and if necessary split into several files.
+
+Ongoing projects
+----------------
+
+Documentation pages for ongoing projects **must** include a :code:`.. warning::` Sphinx directive at the top of the file indicating the code is under development.
+See e.g. :doc:`/transport/index`.
+This section **should** contain one or all of:
+
+- Link(s) to GitHub, including:
+
+  - A current tracking issue, which in turn can link to:
+
+    - Other issues and PRs where work occurs.
+    - Any of the items below.
+
+  - A project board, if any.
+  - A label for issues/PRs, if any.
+
+- Reference to all other locations where work is occurring, including any:
+
+  - Branch(es)—``main``, ``dev``, or any others—in :mod:`message-ix-models` or :mod:`message_data`.
+  - Fork(s) of these repos.
+  - Other repository/-ies separate :mod:`message-ix-models` or :mod:`message_data`
+
+  Not that this **does not** imply those should be made public, for instance prior to publication, if there are reasons not to; only that their existence and contents should be mentioned.
+
+This directive **must** be kept current, and removed once work is complete.
+
+Documentation for ongoing projects **should** be added to :mod:`message_ix_models`, even if some of the code or linked resources are in :mod:`message_data` or are otherwise private.
+
+Completed projects
+------------------
+
+Doc pages for completed projects **must** specify:
+
+- location(s) of scenarios, e.g.
```

Comment:
All other bullet points start with capitalized letters, so I'd do the same here for consistency.

Reply:
Thanks. The general heuristic I use is:

- If the text is formed as sentence with conjunctions (and, or, then) and the part before the list is not a complete sentence, then use lower case, as the bullet list is simply an "unrolled" presentation of the sentence.
- If the bullets are not part of a sentence that starts before or after the list, then capitalized.

I think this instance probably should be the latter, but I've used lower case, so will correct.